apps/console/src/features/organizations/components/edit-organization-role/edit-organization-permission.tsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License
+ */
+
+import { AlertLevels, RolesInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { AppState } from "../../../core";
+import { PermissionList } from "../../../roles";
+import { patchOrganizationRoleDetails } from "../../api";
+import { OrganizationInterface, OrganizationRoleInterface, TreeNode } from "../../models";
+
+/**
+ * Interface to capture permission edit props.
+ */
+interface RolePermissionDetailProps {
+    /**
+     * Role details
+     */
+    roleObject: OrganizationRoleInterface;
+    /**
+     * Show if it is role.
+     */
+    isGroup: boolean;
+    /**
+     * Handle role update callback.
+     */
+    onRoleUpdate: () => void;
+    /**
+     * Show if the user is read only.
+     */
+    isReadOnly?: boolean;
+}
+
+/**
+ * Component to update permissions of the selected role.
```

Comment:
Add new line

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/81488c55dd6c9c7a63575dee78e71f2b80643963